theano/gpuarray/tests/test_cgpukernelbase.py

```diff
@@ -0,0 +1,69 @@
+import numpy
+from six.moves import xrange
+
+import theano
+from theano import tensor, config, Apply
+from theano.gradient import grad_undefined
+
+from .config import mode_with_gpu, test_ctx_name
+
+from ..basic_ops import CGpuKernelBase
+from ..type import GpuArrayType, get_context
+
+
+from pygpu.gpuarray import dtype_to_typecode
+
+
+class TSTGpuEye(CGpuKernelBase):
```

Comment:
use TGpuEye or TestGpuEye. Don't start a new scheme for test name, we already have too many.